sdk/schemaregistry/azure-data-schemaregistry/README.md

```diff
@@ -54,7 +54,7 @@ add the direct dependency to your project as follows.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-data-schemaregistry</artifactId>
-  <version>1.3.1</version>
+  <version>1.3.2</version>
```

Comment:
Update `azure-identity` version too in this README.md.

Reply:
Hmm... I am surprised the prepare-release script didn't update this too.